.github/workflows/ci.yml

```diff
@@ -87,7 +87,8 @@ jobs:
       - name: Run Cargo Trampulin
         uses: actions-rs/tarpaulin@v0.1
         with:
-          args: '--avoid-cfg-tarpaulin --exclude polkadex-node node-polkadex-runtime --workspace --timeout 180'
+          version: '0.22.0'
+          args: '--exclude polkadex-node node-polkadex-runtime --workspace --timeout 180'
```

Comment:
I assume conflict was resolved without leaving `--avoid-cfg-tarpaulin` arg. Will raise PR to dev branch with that fix.

Reply:
Okay thanks

Reply:
[Here](https://github.com/Polkadex-Substrate/Polkadex/pull/630) is a PR.